### PR TITLE
Added back button to compose

### PR DESCRIPTION
**Pull request by joydeepsaha05**
Added the back button to the actionBar to navigate up the stack.

### CODE REVIEW

**Comment by joydeepsaha05** [comment]
#2238

File: k9mail/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -415,11 +419,11 @@
             android:grantUriPermissions="true"
             android:multiprocess="true"
             android:readPermission="${applicationId}.permission.READ_ATTACHMENT">
-
```
Comment (cketti) [inline k9mail/src/main/AndroidManifest.xml]: To avoid accidental unrelated changes such as this one please configure your IDE to not change lines that you haven't modified.

File: k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java [hunk 2]
```diff
@@ -956,6 +959,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.read_receipt:
                 onReadReceipt();
                 break;
+            case android.R.id.home:
+                NavUtils.navigateUpFromSameTask(this);
```
Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java]: This method may only be used if the `MessageList` Activity lives in the same task. But `MessageCompose` can be launched by third-party apps into their own task.

**Comment by cketti** [comment]
We'll go with another pull request for this.
